articles/portal/ptl-ref-release-notes.md

```diff
@@ -19,6 +19,17 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 1.143.0
+
+Date: 7th July 2020
+
+Update and ITHC
```

Comment:
Please use the standard titles. ITHC means nothing to customers. Just use "Feature update". Thanks, Michael

Reply:
Done here: c533fa7